src/bin/pg_autoctl/pgsql.c

```diff
@@ -1563,6 +1581,7 @@ BuildNodesArrayValues(NodeAddressArray *nodeArray,
 			return false;
 		}
 	}
+
 	return paramIndex;
```

Comment:
do we really still need to return `paramIndex`? Also returning both false and paramIndex seems confusing. Let's change the code to actually handle this error.

Reply:
Fixed, thanks for noticing.